### DISCUSSION TITLE
git is not a proper command

### DISCUSSION THREAD

**Discussion by LordSluggo** [carla-simulator/carla#4866] [Q&A]
When I try to run the command " git clone --depth 1 -b carla https://github.com/CarlaUnreal/UnrealEngine.git" I get the error that 

'git' is not recognized as an internal or external command,
operable program or batch file.

I installed everything indicated before this step in the guide and have double-checked everything. Did someone accidentally put some Linux stuff in the Windows guide? I can't do anything to get this to work

**Comment by uservinu** [comment]
reinstall git for windows and make sure add to Path is ticked. Then open a new command prompt. This is not a carla issue
